ledfx/effects/scan.py

```diff
@@ -111,8 +134,15 @@ def render(self):
 
         step_size = step_size * self.bar
 
+        count = self._config["count"]
```

Comment:
I'm a little hazy but I think accessing the config every render is an anti pattern we should be avoiding.

Putting it in the init and config updated functions might be better?

Reply:
Yeah, there is some refactoring here

Reply:
Heavily refactored